etc/kconfigs.yaml

```diff
@@ -127,6 +127,7 @@ IS_SIGNED_TYPE_KUNIT_TEST: v6.1-rc1
 KALLSYMS_SELFTEST: v6.2-rc1
 KASAN: v4.0-rc1, x86_64
 KCSAN_SELFTEST: v5.8-rc1
+KASAN_MODULE_TEST: v5.12-rc1
```

Comment:
help ignore this test, it has too many dependencies that may need more careful check to see any confliction.

Reply:
ok!